distributed/scheduler.py

```diff
@@ -5303,7 +5303,9 @@ def _reevaluate_occupancy_worker(self, ws):
     async def check_worker_ttl(self):
         now = time()
         for ws in self.workers.values():
-            if ws.last_seen < now - self.worker_ttl:
+            if (ws.last_seen < now - self.worker_ttl) and (
+                10 * heartbeat_interval(len(self.workers))
```

Comment:
Terribly confused about this change. Doesn't `10 * heartbeat_interval()` always evaluate to true since `heartbeat_interval` is strictly positive?

Reply:
No, you're right.  i messed up the logic here.